package.json

```diff
@@ -67,6 +67,7 @@
     "react-native-safe-area-context": "^0.6.2",
     "react-native-screens": "^2.0.0-alpha.32",
     "react-native-search-filter": "^0.1.4",
+    "react-native-simple-expand": "^0.1.3",
```

Comment:
**suggestion**: Can we remove this? Doesn't look like you're using it, and we have enough dependencies as it is :)